services/src/main/java/org/fao/geonet/guiservices/util/GetSettingValue.java

```diff
@@ -0,0 +1,39 @@
+package org.fao.geonet.guiservices.util;
+
+import jeeves.interfaces.Service;
+import jeeves.server.ServiceConfig;
+import jeeves.server.context.ServiceContext;
+import org.apache.commons.lang.StringUtils;
+import org.fao.geonet.kernel.setting.SettingManager;
+import org.jdom.Element;
+
+import java.nio.file.Path;
+
+public class GetSettingValue implements Service {
```

Comment:
Would be nice to add some doc on this class, could be useful in the future

Reply:
Done